ogr/services/gitlab/project.py

```diff
@@ -313,6 +313,38 @@ def get_commit_comments(self, commit: str) -> list[CommitComment]:
             for comment in commit_object.comments.list()
         ]
 
+    def get_commit_comment(self, commit_sha: str, comment_id: int) -> CommitComment:
```

Comment:
I'm not sure what's the library for, if we need to do all the dirty work anyways…